python/revng/pypeline/utils/__init__.py

```diff
@@ -2,6 +2,11 @@
 # This file is distributed under the MIT License. See LICENSE.md for details.
 #
 
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from . import cabc, default_dict_from_key, registry  # noqa: F401
```

Comment:
I don't understand why we need these :thinking:

Reply:
That's me trying to get mypy (and other linters) to run on the full codebase. The idea was to force imports of all the modules at type checking time. I'm not 100% sure it's needed tbh